cmd/update.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+)
+
+var updateCmd = &cobra.Command{
+	Use: "update",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 2 {
+			// TODO: Improve this message
+			return fmt.Errorf("Invalid arguments, maximum 1 arguments possible")
+		}
+		return nil
+	},
+	Short: "update the component",
```

Comment:
please add Example
something like this:
```
Example: `  # Change the source of a currently active component to local (use the current directory as a source)
  ocd update --local

  #  Change the source of  frontend component to local with source in ./frontend directory
  ocdev update frontend --local ./frontend

  # Change the source of a currently active component to git 
  ocdev update --git https://github.com/openshift/nodejs-ex.git
	`,
```

Reply:
@kadel Done